src/main.c

```diff
@@ -130,22 +110,24 @@ void AgbMain()
         SetMainCallback2(NULL);
 
     gLinkTransferringData = FALSE;
-    gUnknown_03000000 = 0xFC0;
+    gUnknown_03000000 = sizeof(struct Link);
```

Comment:
where did this come from?

Reply:
We are in a link related function.

Not to mention this is in the beginning of ewram, and the link struct is this size, so I am certain this is where the number came from

Reply:
this is the main loop of the game but ok and that address is very much in IWRAM